gallery/src/guidelines/FontSize/FontSizePage.tsx

```diff
@@ -234,40 +234,40 @@ const FontSizePage = () =>
       <NxP>
         Here is an incomplete list of additional nuances to consider when dealing with inline formatting
       </NxP>
-      <ul className="nx-list nx-list--bulleted">
-        <li className="nx-list__item">
+      <NxList bulleted>
+        <NxList.Item>
           Inline elements do not respond to CSS width, height, margin-top, or margin-bottom properties. Their
```

Comment:
Looks like the `NxList.Text` elements are missing here; you should go ahead and add them

Reply:
> Looks like the `NxList.Text` elements are missing here; you should go ahead and add them

Resolved: https://github.com/sonatype/sonatype-react-shared-components/pull/491/commits/49e847b39ad47fbb73a4d350ad12678101765537